Source/SwiftLintFramework/Rules/SwitchCaseAlignmentRule.swift

```diff
@@ -99,18 +45,132 @@ public struct SwitchCaseAlignmentRule: ASTRule, ConfigurationProviderRule {
 
         let caseLocations = caseStatements.compactMap { caseDict -> Location? in
             guard let byteOffset = caseDict.offset,
-                let (line, char) = contents.lineAndCharacter(forByteOffset: byteOffset) else {
-                    return nil
+                  let (line, char) = contents.lineAndCharacter(forByteOffset: byteOffset) else {
+                return nil
             }
+
             return Location(file: file.path, line: line, character: char)
         }
 
+        guard let firstCase = caseLocations.first,
+              let firstCaseCharacter = firstCase.character else {
+            return []
+        }
+
+        // If indent_cases is on, the first case should be indented from its containing switch.
+        if configuration.indentedCases, firstCaseCharacter <= switchCharacter {
+            return caseLocations.map(locationToViolation)
```

Comment:
do any of the examples trigger this case? Just by looking at them, it doesn't look like it.

Reply:
I believe all of them do as the triggering examples are the non-indented examples. 
what's probably not covered by the tests would be following lines - 

e.g. if the first line is ok but the following cases aren't indented (as it would fall in this condition)